### PR TITLE
Streamline running phpcs and linting and minus-x

### PR DESCRIPTION
**Pull request by paladox**
We move running phpunit from composer test.
Instead in https://github.com/gesinn-it-pub/docker-compose-ci/pull/6 we run composer phpunit.

This is so phpcs now runs.

We removed a phpcs.xml file as we have one under .phpcs.xml. We don't need duplicates.

## Summary by CodeRabbit

- **New Features**
	- Enhanced testing scripts to include linting and code style checks.
- **Bug Fixes**
	- Improved logical clarity in various conditional statements across multiple components.
- **Documentation**
	- Adjusted formatting for better readability in test cases.
- **Chores**
	- Updated commit reference for a subproject.
	- Simplified command for running unit tests in the workflow configuration.
	- Removed redundant step for running unit tests in the workflow.

### CODE REVIEW

**Comment by paladox** [comment]
We can remove rules incrementally from .phpcs.xml. I didn't see the point in having a duplicate file that had different rules.

**Comment by paladox** [comment]
Requires https://github.com/gesinn-it-pub/docker-compose-ci/pull/6 to be merged, I updated the submodule in this change to be able to run it.

**Comment by paladox** [comment]
@coderabbitai review

**Comment by paladox** [comment]
This fixes it @JeroenDeDauw @gesinn-it-gea

**Comment by paladox** [comment]
Could you review this one please @JeroenDeDauw ? :)

will unbreak the tests

File: composer.json [hunk 1]
```diff
@@ -106,7 +106,9 @@
 	},
 	"scripts": {
 		"test": [
```
Comment (gesinn-it-gea) [inline composer.json:108]: sounds strange, that a test target actually does not run tests

Comment (paladox) [reply composer.json:108]: Fixed.

**Comment by paladox** [comment]
@coderabbitai review

**Comment by paladox** [comment]
> @paladox looks good! Ready to merge?

Yup!

**Pull request merged**